app/Services/UpdateAvatarService.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Services;
+
+use Illuminate\Support\Facades\Storage;
+use App\Models\User;
+use Illuminate\Support\Str;
+
+/**
+ * Class UpdateAvatarService.
+ */
+class UpdateAvatarService
+{
+    public static function UpdateAvatarService($avatar, $id)
```

Comment:
tìm hiểu, phân biệt cho anh function có static và không có static khác nhau thế nào?
nêu lý do sao trường hợp này sao cần static nhé

Reply:
theo như em hiểu thì static là biến toàn cục, có nghĩa là nó trả về 1 giá trị trong toàn bộ file xử lí . Còn function không có static trả về giá trị cho mỗi biến xử lí

Reply:
trường hợp này thì không cần static cũng được ạ , vì file này chỉ xử lí một mình nó

Reply:
static function là dạng tĩnh , không cần khởi tạo , và không có tính kế thừa

Reply:
@quandarius-pfp dạng tĩnh là dạng như thế nào khác gì với dạng động?